pom.xml

```diff
@@ -62,9 +67,9 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.pircbotx</groupId>
+            <groupId>com.github.pircbotx</groupId>
             <artifactId>pircbotx</artifactId>
-            <version>2.0.1</version>
+            <version>-2.1-g479835f-165</version>
```

Comment:
Thank you so much for the pircbot update, it kept raising `SEVERE` logs such as:
```

May 17, 2021 11:38:05 AM SEVERE org.pircbotx.PircBotX startLineProcessing
Exception encountered when parsing line
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
```
Due to the outdated version not recognizing some received messages. The spam has vanished as a result of this dependency bump.

Reply:
You're welcome!